src/app/features/suggested-first-steps/suggested-first-steps.tsx

```diff
@@ -13,21 +16,52 @@ import { ExploreAppsStep } from './components/explore-apps-step';
 import { BuyNftStep } from './components/buy-nft-step';
 
 export function SuggestedFirstSteps() {
+  const [hasCompletedBackUpSecretKeyStep, setHasCompletedBackUpSecretKeyStep] = useState(true);
+  const [hasCompletedAddFundsStep, setHasCompletedAddFundsStep] = useState(false);
```

Comment:
So, say we killed the `onComplete` for the child component of this step.

Wouldn't it be simpler to do the logic here?

```ts
const hasCompletedAddFundsStep = availableBalance.isGreaterThan(0);
```